### PR TITLE
Hotfix/load ldat plot

### PR DESCRIPTION
**Pull request by zackorenberg**
Even though I am still working on converting all the python2 BlueFors scripts, this hotfix allows the converted BlueFors logs to be usable in LabGUI, including with SynchronizeData.py, which can be useful for data being taken currently.

### CODE REVIEW

File: LabTools/Display/PlotDisplayWindow.py [hunk 1]
```diff
@@ -564,8 +564,8 @@ def YCheckBoxHandler(self):
         """Update which data is used for the Y axis (both left and right)"""
         if self.chan_X is None:
             return
-        else:
-            print(self.chan_X)
+        #else:
```
Comment (Bachibouzouk) [inline LabTools/Display/PlotDisplayWindow.py:567]: If you think it is not useful, then you can delete it rather than commenting it ;)

File: tools/SynchronizeData.py [hunk 2]
```diff
@@ -97,6 +117,8 @@ def __init__(self, file="", *args, **kwargs):
 
             self.raw_data = np.loadtxt(file)
         else:
+            if extension == "ldat":
+                print("ldat library missing")
```
Comment (Bachibouzouk) [inline tools/SynchronizeData.py:121]: maybe raise a warns.Warning() ?

File: tools/SynchronizeData.py [hunk 3]
```diff
@@ -610,10 +644,14 @@ def save(self, header, data, output_file=None):
 
 if __name__ == "__main__":
```
Comment (Bachibouzouk) [inline tools/SynchronizeData.py:645]: I would not include the changes made here if they contain files path on a specific computer. In general, I would not commit changes made under the `if __name__ == "__main__"`

**Review by Bachibouzouk — APPROVED** [review]
I don't have the time to test it locally but it seems cool.

One thing you could/should do is to write tests whenever you add a feature or fix a problem (here the test could be to load a dummy file and check that the markers are not None for example), it is much lighter to write tests as you go compare to suddenly have to write a lot of tests. And it is also good to have tests to make sure a change will not hinder the existing functionalities (creating some bugs that may be difficult to track down, especially if the bug is noticed only after you introduces multiple changes and you cannot be 100% which of the change created the bug)
